Readme.md

```diff
@@ -1,13 +1,14 @@
 
 # escape-html
 
-  Escape HTML entities
+  Escape HTML special characters (syntax), replacing them with character references.
 
 ## Example
 
 ```js
 var escape = require('escape-html');
-escape(str);
+escape('<meta content="\'a\' & b" />');
+// &lt;meta content=&quot;&#39;a&#39; &amp; b&quot; /&gt;
```

Comment:
> I feel like this is too noisy and a really weird thing for someone to actually want to escape?

Yeah, it's just a contrived example to illustrate all of the characters that will currently be escaped. I guess something like this would be better, to avoid the noise of having to escape the single quotes for JS:

```js
escape('<p title="A&B">');
// &lt;p title=&quot;A&amp;B&quot;&gt;

escape("<p title='A&B'>");
// &lt;p title=&#39;A&amp;B&#39;&gt;
```

> Especially with the trailing `/` which is XHTML syntax.

It's [just HTML](https://html.spec.whatwg.org/multipage/syntax.html#start-tags). But it's also an irrelevant style thing -- omitting it from the example would be fine.